### PR TITLE
[UXIT-1808] Remove and/or replace no response links

### PR DESCRIPTION
**Pull request by barbaraperic**
## 📝 Description

[Resource](https://docs.google.com/spreadsheets/d/1DEf09aZeM4C7M2ItmBXE46q9nAEskDE9bpnzGAMB-eE/edit?gid=20143518#gid=20143518)

- KenLabs: Appears to no longer exist. Their website shows a 500 error, and both their [GitHub](https://github.com/kenlabs) and [Twitter](https://x.com/KenLabs_Web3) accounts have been deleted.

- LabDao: Updated the broken link to the main URL from their [Twitter](https://x.com/lab_dao).

- LurkLang: Now appears to be part of [Augment Computer](https://github.com/lurk-lab). Since it no longer operates as a standalone entity, I've removed it.

- Bytebase.cn: No longer seems to exist.

### CODE REVIEW

File: src/content/blog/storage-company-ewesion-leverages-filecoin-network-for-data-preservation-1.md
Comment (mirhamasala) [inline src/content/blog/storage-company-ewesion-leverages-filecoin-network-for-data-preservation-1.md:1]: I would keep the text but remove the link

File: src/content/blog/storage-company-ewesion-leverages-filecoin-network-for-data-preservation.md
Comment (mirhamasala) [inline src/content/blog/storage-company-ewesion-leverages-filecoin-network-for-data-preservation.md:1]: Same!

File: src/content/blog/xinshu-technology-how-the-chinese-shutterfly-uses-the-filecoin-network-to-keep-a-competitive-edge-in-the-photo-sharing-industry.md
Comment (mirhamasala) [inline src/content/blog/xinshu-technology-how-the-chinese-shutterfly-uses-the-filecoin-network-to-keep-a-competitive-edge-in-the-photo-sharing-industry.md:1]: Same!

File: src/content/ecosystem-explorer/projects/ken-labs.md
Comment (mirhamasala) [inline src/content/ecosystem-explorer/projects/ken-labs.md:1]: We can’t remove these outright, as doing so would result in a 404 error. Let’s consult David on the best approach. Perhaps we could implement a redirect to /ecosystem-explorer.

File: src/content/ecosystem-explorer/projects/lurk-lab.md
Comment (mirhamasala) [inline src/content/ecosystem-explorer/projects/lurk-lab.md:1]: We can’t remove these outright, as doing so would result in a 404 error. Let’s consult David on the best approach. Perhaps we could implement a redirect to /ecosystem-explorer.

**Review by mirhamasala — CHANGES_REQUESTED** [review]
We need to make some changes and look into what to do with non-existing ecosystem projects. 🙏🏼

File: src/content/ecosystem-explorer/projects/ken-labs.md
Comment (barbaraperic) [reply src/content/ecosystem-explorer/projects/ken-labs.md:1]: How about using a label similar to GitHub's approach for unmaintained repositories, such as "Archived"?

Comment (mirhamasala) [reply src/content/ecosystem-explorer/projects/ken-labs.md:1]: Yes, that could be possible. I added a ticket here [>>](https://www.notion.so/filecoin/FF-Think-of-what-to-do-with-inactive-Ecosystem-Projects-1537631f282580f1837cce5574097301?pvs=4)

Comment (mirhamasala) [reply src/content/ecosystem-explorer/projects/ken-labs.md:1]: I was a bit surprised to find that we could just remove website - We seem to have an inconsistency with config (where it's required) and Zod Schema (where it's optional).

**Review by mirhamasala — APPROVED** [review]
Thank you! LGTM! ✨

**Pull request merged**